src/reporting.js

```diff
@@ -215,39 +216,52 @@ async function _generateReportFromDownload(req, domain, mediaId, matrixFile, opt
     console.info(`Downloading ${httpUrl}, writing to ${filePath}`);
 
     let downloadHeaders;
-    let response;
 
     try {
         downloadHeaders = await new Promise((resolve, reject) => {
-            let responseHeaders;
-	    let connect = {url: httpUrl, encoding: null, qs: thumbnailQueryParams};
-	    const config = getConfig();
-	    const proxy = config.proxy;
-	    if (proxy != null) {
-		connect.proxy = proxy;
-	    }
-	    if (config.requestHeader != null) {
-		connect.headers = generateRequestHeaders(config.requestHeader, req);
-		console.info(`Request headers are`, connect.headers);
```

Comment:
I also switched this from a `console.info` to `console.debug` as I don't think it needs to be at the info level...

This line was added recently in dae7bcd51227f32123fbd181738058fdd62ce1e9.